lib/lib.sh

```diff
@@ -95,13 +95,9 @@ get_meta() {
             echo "$meta"
             return 0
         fi
-        if [ ! -v EC2_IF_INITIAL_SETUP ]; then
-            return 1
-        else
-            attempts+=1
-            backoff=$((attempts*ms_per_backoff))
-            sleep $((backoff/1000)).$((backoff%1000))
-        fi
+        attempts+=1
+        backoff=$((attempts*ms_per_backoff))
+        sleep $((backoff/1000)).$((backoff%1000))
```

Comment:
Brittle code alert: This works only because `ms_per_backoff` is 100. You can use printf to do the leading 0's as expected:
```
$ backoff=1100
$ echo $((backoff/1000)).$((backoff%1000))
1.100
$ backoff=1010
$ echo $((backoff/1000)).$((backoff%1000))
1.10
$ echo $(printf "%u.%03u" $((backoff/1000)) $((backoff%1000)))
1.010
$ 
```